src/applications/medical-expense-report/config/form.js

```diff
@@ -40,8 +38,8 @@ const formConfig = {
   //   dependencies: [externalServices.icmhs],
   // },
   formId: VA_FORM_IDS.FORM_21P_8416,
-  // useCustomScrollAndFocus: false,
-  defaultDefinitions: commonDefinitions,
+  useCustomScrollAndFocus: false,
```

Comment:
Can you re comment this out, I can't remember if it was causing issues with the international phone number either.